docs/en/deployment/hadoop_java_sdk.md

```diff
@@ -204,6 +204,7 @@ Please refer to the following table to set the relevant parameters of the JuiceF
 | `juicefs.push-auth`       |               | [Prometheus basic auth](https://prometheus.io/docs/guides/basic-auth) information, format is `<username>:<password>`.                                                       |
 | `juicefs.push-graphite`   |               | [Graphite](https://graphiteapp.org) address, format is `<host>:<port>`.                                                                                                     |
 | `juicefs.push-interval`   | 10            | Metric push interval (in seconds)                                                                                                                                           |
+| `juicefs.push-labels`     |               | Metric labels, format is `<key1>:<value1>,<key2>:<value2>`.                                    |
```

Comment:
`key1:value1,key2:value2`  
This format is simple and straightforward and will not be misinterpreted

Reply:
so you mean just delete this part of doc?

Reply:
just replace `<key1>:<value1>,<key2>:<value2>` to `key1:value1,key2:value2`

Reply:
Like this.
<img width="839" alt="image" src="https://github.com/juicedata/juicefs/assets/31313340/71cd8c9f-8d7f-438c-abb1-552ca603438a">